Cartfile

```diff
@@ -4,6 +4,7 @@ github "ReactiveX/RxSwift"
 github "RxSwiftCommunity/RxDataSources" ~> 3.0
 github "RxSwiftCommunity/RxOptional" ~> 3.1.3
 github "mozilla-mobile/telemetry-ios"
-github "mozilla-lockbox/lockbox-ios-fxa-sync"
 github "jrendel/SwiftKeychainWrapper" ~> 3.0
-github "mozilla-mobile/MappaMundi" "master"
\ No newline at end of file
+github "mozilla-mobile/MappaMundi" "master"
+github "mozilla/application-services" "0.2.0"
+github "mozilla-lockbox/lockbox-ios-fxa-sync" "044d84bfbe44e6fb43e4b7737cf551ca77a918da"
```

Comment:
Reminder to merge https://github.com/mozilla-lockbox/lockbox-ios-fxa-sync/pull/10 and change this back to no SHA or at the latest SHA, whichever is preferred.